src/atoms/tile/AppTile.ts

```diff
@@ -0,0 +1,41 @@
+import { TTile } from '../../types';
```

Comment:
Don't store all types in one basket, better to have them near the place they needed, in this case in this exact folder. Try to think about the folder as "all the context I need to understand this atom"

Reply:
It is done in such way because it was said during practice lessons. And also according to the structure in example of atomic architecture from lessons